storage/remote/write.go

```diff
@@ -37,6 +37,12 @@ var (
 		Name:      "samples_in_total",
 		Help:      "Samples in to remote storage, compare to samples out for queue managers.",
 	})
+	exemplarsIn = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "exemplars_in_total",
+		Help:      "Exemplars in to remote storage", //, compare to samples out for queue managers.",
```

Comment:
Yea, I don't understand this `compare to samples out for queue managers.",` on sample series either.. wonder can we ask someone what that means? 🙃

Reply:
Not sure why this part of the help text is commented out, it should be included :smile: this is essentially saying 'compare `exemplars_in_total` to the samples out metric from a queue manager to know if/how far behind that queue is.